RiotSwiftUI/Modules/Room/LocationSharing/Coordinator/ShareToMapsAppActivity.swift

```diff
@@ -20,25 +20,25 @@ extension UIActivity.ActivityType {
     static let shareToMapsApp = UIActivity.ActivityType("Element.ShareToMapsApp")
 }
 
-class ShareToMapsAppActivity: UIActivity {
-    enum MapsAppType {
+public class ShareToMapsAppActivity: UIActivity {
```

Comment:
Just curious, how come this needs to be public now?

Reply:
Good question. I think I initially wanted to build them directly from the RoomViewController but then decided against it and forgot to change them back.(or intentionally didn't in case we need them? 🤔 )
Either way I'll change them back.